libs/utils/src/lib/helpers/functions.ts

```diff
@@ -937,3 +937,12 @@ export function findAncestorElement(
 export function hasInvalidChars(str: string, invalidChars: string[]): boolean {
   return invalidChars.some((invalidChar) => includes(str, invalidChar));
 }
+
+/**
+ * Returns the closest thing to a "name" on an object.
+ * @arg {any} obj - An object to get the name of.
+ * @return {string} - The name of the object.
+ */
+export function getObjectName(obj: any): string {
+  return obj?.displayName ?? obj?.properties?.displayName ?? obj?.name ?? obj?.properties?.name ?? obj?.id ?? '';
```

Comment:
Why do we want to show a resource display name when selecting a resource.... the Name column specifically means resource name, if we want to show a friendly name it should be a separate column..
And dont call it object since it looks like we are looking for objectKey like json object key